contracts/test/unit/oracle/VoltSystemOracle.t.sol

```diff
@@ -75,40 +75,24 @@ contract VoltSystemOracleUnitTest is DSTest {
         assertEq(voltSystemOracle.getCurrentOraclePrice(), expectedOraclePrice);
     }
 
-    function testLERPOneQuarter() public {
-        _testLERP(
-            block.timestamp + voltSystemOracle.periodStartTime() + 91.25 days
-        );
-    }
-
-    function testLERPHalfYear() public {
-        _testLERP(
-            block.timestamp + voltSystemOracle.periodStartTime() + 182.5 days
-        );
-    }
-
-    function testLERPThreeQuarters() public {
-        _testLERP(
-            block.timestamp + voltSystemOracle.periodStartTime() + 273.75 days
-        );
-    }
-
     function testLERPPerDay() public {
         vm.warp(block.timestamp + voltSystemOracle.periodStartTime());
-        for (uint256 i = 1; i < 363; i++) {
+        for (uint256 i = 1; i < voltSystemOracle.TIMEFRAME() / 1 days; i++) {
```

Comment:
curious as to why we divide by 1 day here?

Reply:
time frame = 30.42 days * seconds per day, so we need to divide time frame by seconds per day to only calculate price changes over a month